packages/flutter/lib/src/painting/matrix_utils.dart

```diff
@@ -127,6 +127,13 @@ class MatrixUtils {
   ///
   /// This function assumes the given point has a z-coordinate of 0.0. The
   /// z-coordinate of the result is ignored.
+  ///
+  /// While not common, this method may return (NaN, NaN), iff the given `point`
```

Comment:
Just checking - 'iff' as in 'if and only if', or typo?

Reply:
Yeah I meant "if and only if".